.github/workflows/test-e2e.yml

```diff
@@ -144,11 +144,20 @@ jobs:
           done
       - name: Checkout template-operator
         uses: actions/checkout@v3
-        if: ${{ matrix.e2e-test == 'module-status-propagation' }}
+        if: ${{ matrix.e2e-test == 'module-status-propagation' || matrix.e2e-test == 'non-blocking-deletion' }}
         with:
           repository: kyma-project/template-operator
           path: ./template-operator/
       - name: Create Template Operator Module
+        working-directory: ./template-operator
+        if: ${{ matrix.e2e-test == 'non-blocking-deletion' }}
+        run: |
+          kyma alpha create module --kubebuilder-project  --channel=regular --name kyma.project.io/module/template-operator --version 1.1.1 --path . --registry k3d-kcp-registry:5111 --insecure --module-archive-version-overwrite /
```

Comment:
please change the create module to module config file instead of the deprecated kubebuilder-project. 

you can have a reference how it's doing in this [PR](https://github.com/kyma-project/lifecycle-manager/pull/911/files#diff-976614657a9537dd6476a2977a4a11c0ac912e262006a4ca2d4fe0b4d8cf0becR179)
or in [CLI](https://github.com/kyma-project/cli/blob/main/.github/workflows/test-e2e-create-module.yml#L74-L81)